### PR TITLE
Support icelandic

### PR DESCRIPTION
**Pull request by jamiees2**

### CODE REVIEW

**Comment by HermannBjorgvin** [comment]
I support this pull request as an Iceland who's always felt ignored and left out in this project. I can also vouch for @jamiees2 spelling being correct.

Vér mótmælum allir!

**Comment by Joezo** [comment]
If you add to the tests i'll merge this :)

**Comment by jamiees2** [comment]
No problem! Here you go :3

File: five.js [hunk 1]
```diff
@@ -44,6 +44,7 @@
   five.indonesian = function() { return 'lima'; };
   five.irish = function() { return 'cúig'; };
   five.italian = function() { return 'cinque'; };
+  five.icelandic = function() { return 'fimm'; };
```
Comment (Joezo) [inline five.js]: Can you move this up above `indonesian` so that it maintains alphabetical order? :) same in the test file and readme

**Comment by jamiees2** [comment]
I updated the order :)

**Comment by Joezo** [comment]
Thanks :D

**Pull request merged**